.buildkite/build_cache/setup_ci_build_cache.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+# The script should immediately exit if any command in the script fails.
+set -e
+
+SCRIPT=$(readlink -f "$0")
+DIR=$(dirname "$SCRIPT")
+
+bazel_rc="${DIR}/../bazel.rc"
+
+PULL_BRANCH_REGEX="^pull\/[0-9]+$"
+
+# Forked pull requests have CIRCLE_BRANCH set to pull/XXX
```

Comment:
It might be a really dumb question - but how is buildkite connected to CIRCLE_BRANCH naming?

Reply:
True, this is a leftover. thx for the hint